Automattic/vip-go-mu-plugins
# Fix imports when Jetpack is disabled

## Description

We allow Jetpack to be disabled through `VIP_JETPACK_SKIP_LOAD`. If that is the case, site imports through the CLI will fail, because in their cleanup phase, they try to reconnect Jetpack.

This PR checks if Jetpack is disabled, and if that's the case, it skips that reconnection attempt.

## Changelog Description

### Fix imports when Jetpack is disabled

Imports through CLI would fail if Jetpack is disabled on the site.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Disable Jetpack through `VIP_JETPACK_SKIP_LOAD`
3. Perform a site import. It should complete
